src/components/login.tsx

```diff
@@ -0,0 +1,63 @@
+"use client"
+import { FormEvent } from "react"
+import { useState } from "react"
+import crypto from "crypto"
+import getPBKDF2DerivedKey from "@/utils/getPBKDF2DerivedKey"
+
+export default function Login() {
+
+    const [isLogged, setLogged] = useState(false)
```

Comment:
It could have been a great Idea, but the fact is that the parent is a server component, so it's not possible to do so, a better idea would be to store it in a localStorage or something else a bit better.